### PR TITLE
[JENKINS-28343] Add quiet option to p4-plugin sync

### PR DESCRIPTION
**Pull request by rpocase**
Adding option to Populate implementations to use -q flag for syncing.

1) Adding checkboxes and help text for quiet option
2) Adding quiet to Populate constructors
3) Setting quiet option where necessary

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/p4/client/ClientHelper.java [hunk 1]
```diff
@@ -130,7 +130,6 @@ public boolean updateFiles() throws Exception {
 
 		// Sync revision to re-edit
 		SyncOptions syncOpts = new SyncOptions();
-		syncOpts.setNoUpdate(true);
```
Comment (p4paul) [inline src/main/java/org/jenkinsci/plugins/p4/client/ClientHelper.java]: Why did you remove the sync '-n' flag?  This is used to run a preview sync to see if the workspace is up-to-date.

File: src/main/resources/org/jenkinsci/plugins/p4/populate/AutoCleanImpl/config.jelly [hunk 2]
```diff
@@ -11,6 +11,10 @@
 	<f:entry field="modtime">
 		<f:checkbox title="${%Sync with MODTIME for consistency check}" default="false"/>
 	</f:entry>
+
+	<f:entry field="quiet">
+		<f:checkbox title="${%Suppress Perforce generated info messages}" default="false"/>
```
Comment (p4paul) [inline src/main/resources/org/jenkinsci/plugins/p4/populate/AutoCleanImpl/config.jelly]: Perhaps a default of 'true'?
The current release 1.2.2 is very quiet.  It is only the recent changes in Main that added the Perforce output to the console log.

File: src/test/java/org/jenkinsci/plugins/p4/PerforceScmTest.java [hunk 3]
```diff
@@ -24,7 +24,7 @@ public void testConfigBasic() throws Exception {
 
 		String credential = "123";
 		Workspace workspace = new StaticWorkspaceImpl("none", false, "test.ws");
-		Populate populate = new AutoCleanImpl(true, true, false, null);
+		Populate populate = new AutoCleanImpl(true, true, false, false, null);
```
Comment (p4paul) [inline src/test/java/org/jenkinsci/plugins/p4/PerforceScmTest.java]: For the tests you can leave these 'false'.

File: src/main/java/org/jenkinsci/plugins/p4/client/ClientHelper.java (hunk 1 above)
Comment (rpocase) [inline src/main/java/org/jenkinsci/plugins/p4/client/ClientHelper.java]: Entirely accidental. I'll add it back.

File: src/main/resources/org/jenkinsci/plugins/p4/populate/AutoCleanImpl/config.jelly (hunk 2 above)
Comment (rpocase) [inline src/main/resources/org/jenkinsci/plugins/p4/populate/AutoCleanImpl/config.jelly]: Sure. I'll change the default to true.

File: src/test/java/org/jenkinsci/plugins/p4/PerforceScmTest.java (hunk 3 above)
Comment (rpocase) [inline src/test/java/org/jenkinsci/plugins/p4/PerforceScmTest.java]: I left all original booleans as is - just added a "false" parameter for the new parameter.
